rustls/src/conn.rs

```diff
@@ -791,6 +791,8 @@ impl<Data> ConnectionCore<Data> {
                 .common_state
                 .has_received_close_notify
             {
+                // "Any data received after a closure alert has been received MUST be ignored."
```

Comment:
Feel free to squash. 👍

Reply:
Done -- also reworded the commit message.